Packs/QRadar/Integrations/QRadar_v2/README.md

```diff
@@ -1353,3 +1353,20 @@ There is no context output for this command.
 #### Human Readable Output
 
 >fetch-incidents was reset successfully.
+
+### get-mapping-fields
+***
+Returns the list of fields for an incident type. This command is for debugging purposes.
+This command is used in [Classification & Mapping](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html) section.
```

Comment:
please move this line and the one below it to the `Fetch incidents` section in this README under the title `#### Field Mapping`
need to change it a bit also, to something like:
```
The integration support mapping fields based on schema.
The scheme is divided to 4 sections. Offense (root), Events: Builtins, Events: Custom Fields and Assets.
For more details, see the [Classification & Mapping documentation](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/incidents/classification-and-mapping/create-a-mapper.html).
```

Reply:
DONE APPROVE PLEASE